integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -353,4 +353,12 @@ public interface TestConstants {
       + "config/samples/sidb/singleinstancedatabase.yaml";
   public static final String ORACLELINUX_TEST_VERSION =
       getNonEmptySystemProperty("wko.it.oraclelinux.test.version", "7");
+
+  //retry improvement
```

Comment:
Shouldn't we add a integration test to verify the runtime behavior of these new attributes.

Reply:
we have OWLS-101201 and OWLS-101206 to add integration tests to test detailed of the improvements. In this PR, we adding these two options that are used by a lot of test suites, important, the default values of the options are changed to a very small value to trigger any issues in case